osu.Game.Rulesets.Osu.Tests/TestSceneOsuAnalysisContainer.cs

```diff
@@ -26,6 +27,8 @@ public partial class TestSceneOsuAnalysisContainer : OsuTestScene
         [Cached]
         private OsuRulesetConfigManager config = new OsuRulesetConfigManager(null, new OsuRuleset().RulesetInfo);
 
+        private readonly StopwatchClock clock = new StopwatchClock();
```

Comment:
I haven't checked on this yet, but in cases like this we usually use `ManualClock`. Is there a specific reason we're using `StopwatchClock` here?

Reply:
You can't stop or start it which seemed counterproductive to being able to see what the thing looks while time is progressing.